src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,47 @@
+package lotto.domain;
+
+import lotto.exception.InvalidInputException;
+import lotto.strategy.GenerateLottoNumber;
+
+import java.util.Set;
+import java.util.TreeSet;
+
+public class Lotto {
+    private static final int SIZE = 6;
+    private static final String INVALID_SIZE_OF_WINNING_NUMBERS = "번호의 갯수가 올바르지 않습니다.";
+
+    private Set<LottoNumber> numbers;
+
+    public Lotto() {
+        numbers = new TreeSet<>();
+        while (numbers.size() < SIZE) {
+            numbers.add(new LottoNumber(GenerateLottoNumber.generateRandomNumbers()));
+        }
+    }
+
+    public Lotto(String strNumbers) {
+        numbers = new TreeSet<>();
+        for (String strNumber : strNumbers.replaceAll(" ", "").split(",")) {
+            numbers.add(new LottoNumber(Integer.parseInt(strNumber)));
+        }
```

Comment:
```suggestion
        for (String strNumber : strNumbers..split(",")) {
            numbers.add(new LottoNumber(strNumber));
        }
```
이렇게 표현할 수 있도록 LottoNumber를 만들어주면 어떨까요?
```java
LottoNumber.valueOf("1 ");
LottoNumber.valueOf(" 1 ");
LottoNumber.valueOf(" 1");
LottoNumber.valueOf(1);
```
이렇게 4개 모두 동일한 instance를 반환하도록 작성해보세요!~